ver/ique/splat.yaml

```diff
@@ -377,7 +403,13 @@ segments:
     - [0x729B0, .rodata, audio/reverb]
     - [0x72A10, .rodata, 5B320]
     - [0x72A80, .rodata, world/world]
-    - [0x73B10, rodata, libultra]
+    - [0x73B10, .rodata, os/lookathil]
```

Comment:
The entire .rodata list can be removed and you can just end change the end segment marker to the first .rodata section. so like after the last .data, just do - [0x71DB0]